imap-codec/src/codec/encode.rs

```diff
@@ -1171,6 +1180,21 @@ impl<'a> EncodeIntoContext for Code<'a> {
                 ctx.write_all(b"REFERRAL ")?;
                 ctx.write_all(url.as_bytes())
             }
+
+            // RFC 4551
+            #[cfg(feature = "ext_condstore_qresync")]
+            Code::HighestModSeq(modseq) => {
+                ctx.write_all(b"HIGHESTMODSEQ ")?;
+                modseq.encode_ctx(ctx)
+            }
+            #[cfg(feature = "ext_condstore_qresync")]
+            Code::NoModSeq => ctx.write_all(b"NOMODSEQ"),
+            #[cfg(feature = "ext_condstore_qresync")]
+            Code::Modified(seq_or_uid_list) => {
+                ctx.write_all(b"MODIFIED ")?;
+                join_serializable(seq_or_uid_list.as_ref(), b",", ctx)
```

Comment:
Here, I stopped for a moment ...

The RFC tells ...

```abnf
resp-text-code      =/ "HIGHESTMODSEQ" SP mod-sequence-value /
                       "NOMODSEQ" /
                       "MODIFIED" SP set
```

... w/o saying what `set` is. Do you have more information on that?

Reply:
Hm... so I guess it should be [`SequenceSet`](https://docs.rs/imap-types/latest/imap_types/sequence/struct.SequenceSet.html) instead of `Vec1<...>`?